src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -286,12 +303,25 @@ namespace AirflowNetwork {
         bool TExtHeightDep;          // Choice of height dependence of external node temperature
         bool AllowSupportZoneEqp;    // Allow unsupported zone equipment
         // "ZeroNodePressures", or "LinearInitializationMethod"
+        bool AFNDuctAutoSize;          // True: perform duct autosize, otherwise no duct autosize
+        DuctSizeMethod ductSizeMethod; // Duct autosize method as enum
+        Real64 DuctSizeFactor;         // Duct size factor
+        Real64 DuctSizeMaxV;           // Maximum airflow velocity
+        Real64 DuctSizePLossSTrunk;    // Pressure loss across supply trunk
+        Real64 DuctSizePLossSBranch;   // Pressure loss across supply branch
+        Real64 DuctSizePLossRTrunk;    // Pressure loss across return trunk
+        Real64 DuctSizePLossRBranch;   // Pressure loss across return branch
+        int ErrCountDuct;
+        int ErrIndexDuct;
 
         // Default Constructor
         AirflowNetworkSimuProp()
             : Control("NoMultizoneOrDistribution"), WPCCntr("Input"), MaxIteration(500), InitFlag(0), solver(Solver::SkylineLU), RelTol(1.0e-5),
               AbsTol(1.0e-5), ConvLimit(-0.5), MaxPressure(500.0), Azimuth(0.0), AspectRatio(1.0), DiffP(1.0e-4), ExtLargeOpeningErrCount(0),
```

Comment:
Are things like `AbsTol`, `RelTol`, `ConvLimit` etc. constants or actual variables that can change from run to run or even within a run? If constants, pull them out of the object and make them `constexpr`. No need to construct and load values that the compiler can hardcode.

Reply:
Those are user inputs and are necessary because there are (hopefully infrequently) cases in which convergence can be an issue.

Reply:
The comments are related to @jasondegraw PR. I will leave them for the time being.